docs/labwc-config.5.scd

```diff
@@ -77,6 +77,10 @@ The rest of this man page describes configuration options.
 	be used with labwc the preferred mode of the monitor is used instead.
 	Default is no.
 
+*<core><cycleViewOSD>* [yes|no]
+	Draw the OSD when cycling between windows.
```

Comment:
Maybe s/OSD/window menu/ or "window dialog" or something similar?

Reply:
I'd vote for:
`Draw the window switcher overlay when cycling between windows.`

Reply:
Hm.. you are right. Just had a look myself and especially `labwc-config.5.scd` has some more references to `OSD`.
Still not really happy with that but it can be changed for all the occurrences via a separate PR.